infrastructure/modules/vpc/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
+  num_az_zones = length(var.azs)
+  az_zones     = var.azs
+}
+
+locals {
+  public_subnet_cidrs  = [for i in range(1, local.num_az_zones + 1) : "10.0.${i}.0/24"]
+  private_subnet_cidrs = [for i in range(1, local.num_az_zones + 1) : "10.0.10${i}.0/24"]
+}
```

Comment:
do you just want one public and one private subnet per AZ zone? could create other variables called num public and num private subnets along with num az zones and spread them out across the AZ zones. 

Alternatively, you could pull the existing VPC module from the Answer digital terraform repo. We are trying to encourage people to use Answers terraform modules more, but its up to you :)

Reply:
sorry I may have been confusing by commenting on the locals.tf , what I meant was these extra variables should be added in vpc/variable.tf so that the user can specify how many az zones they want along with how many private and public subnets they want e.g. 2 AZ zones with 2 private subnets and 2 public subnets in each AZ zone ( total of 4 public and 4 private subnets)